package/MDAnalysis/lib/util.py

```diff
@@ -374,7 +375,9 @@ def _get_stream(filename, openfunction=open, mode='r'):
     """Return open stream if *filename* can be opened with *openfunction* or else ``None``."""
     try:
         stream = openfunction(filename, mode=mode)
-    except IOError:
+    except (IOError, OSError) as err:
```

Comment:
Ok it's all a little confusing how this is structured, but I think this is ok...